gapis/gfxapi/vulkan/vulkan.api

```diff
@@ -5496,6 +5496,26 @@ cmd void vkCmdSetStencilReference(
     u32                reference) {
 }
 
+@internal
+class CmdBindDescriptorSets {
+  // map from binding to set
+  map!(u32, ref!DescriptorSetObject) DescriptorSets
+  map!(u32, BoundBuffer)             BoundBuffers
+}
+
+sub void doCmdBindDescriptorSets(CmdBindDescriptorSets bind) {
+  for _ , _ , v in bind.BoundBuffers {
+    if (v.Buffer != null) {
+      v.Buffer.LastBoundQueue = LastBoundQueue
+      readCoherentMemoryInBuffer(v.Buffer, v.Offset, v.Range)
+    }
+  }
+  clearLastDrawInfoDescriptorSets()
```

Comment:
Removed the call to ClearDescriptorSets in 6957177.

The clear function is kept for future 'correct' logic of clearing bound descriptor sets.